src/sentry/migrations/0340_reconstrain_savedsearch_pinning_fields.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 2.2.28 on 2022-11-02 01:54
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not. Generally
+    # we don't want to run in a transaction here, since for long running operations like data
+    # back-fills this results in us locking an increasing number of rows until we finally commit.
+    atomic = False
```

Comment:
This needs to be set to `True` since we're adding a concurrent index. I will update this once #40874 is merged.

Reply:
No, this should remain false. Tbh I should just remove this option from the template, basically there's not much reason to ever set this to true

Reply:
Ahh, I read this

> Note that when CONCURRENTLY is used we can't run the migration in a transaction, so it's important to use atomic = False to run these.

And assumed because it was being called out that it was saying it needed to be set to whatever the opposite of the default value was.

Reply:
Okay, this is back to `False` now ✅